crates/ra_syntax/src/ast/tokens.rs

```diff
@@ -48,6 +48,7 @@ pub enum CommentPlacement {
 const COMMENT_PREFIX_TO_KIND: &[(&str, CommentKind)] = {
     use {CommentPlacement::*, CommentShape::*};
     &[
+        ("////", CommentKind { shape: Line, doc: None }),
```

Comment:
Is `////` specifically called out in rust? If not I think the code that returns the text should be changes instead.

Reply:
Yeah, it explicitly says a line comment begins with either `//` or `////`: https://doc.rust-lang.org/reference/comments.html
![image](https://user-images.githubusercontent.com/16367467/80528378-1f36b480-8996-11ea-8e9e-9df6455cd5a0.png)

Reply:
TIL!

Reply:
Me too!